defaults/main.yml

```diff
@@ -152,5 +152,5 @@ rhel7cis_syslog: rsyslog
 #rhel7cis_syslog: syslog-ng
 
 rhel7cis_vartmp:
-    source: /dev/mapper/rootvg-vtmplv
-    fstype: xfs
+    #source: /dev/mapper/rootvg-vtmplv
+    #fstype: xfs
```

Comment:
Lets set these to sane defaults and bind mount /var/tmp to /tmp. Also add a var for mount options?

```
rhel7cis_vartmp:
    source: /tmp
    fstype: none
    opts: "defaults, nodev, nosuid, noexec, bind"
```

I think that should work.

Reply:
I always agree with sane defaults , but the reason I commented things out here (and added the when: condition) is so the user will have to set a value before this item can ever mess w/their /tmp mount.

In other words, they might have handled /tmp manually and not necessarily by using a bind mount and they may not have seen (thus tweaked) this variable option.

Let me know what you think.

Reply:
I see your point. I'm trying to think of a good way to do this without having the variable parameters commented out though. Maybe just key off of the `source` parameter. ie. If `rhel7cis_vartmp['source'] is None or an empty string` then skip the play since I think having no source attribute is basically an invalid config of this item.

This way the default will be something like:

```
rhel7cis_vartmp:
    source: ''
    fstype: none
    opts: "defaults, nodev, nosuid, noexec, bind"
```
I don't know if that makes sense or is better then what you had though :)

Reply:
rhel7cis_vartmp:
    source: /tmp
    fstype: none
    opts: "defaults, nodev, nosuid, noexec, bind"
    enabled: no

then check enabled?

Reply:
LGTM!